distribution/docs/src/main/resources/_contents/configuring-from-configuration-files-contents.adoc

```diff
@@ -334,7 +334,7 @@ The following table shows the format to use for each property type supported.
 
 |Lists of Strings
 |`name = [ "value1", "value2", ... ]`
-|`authenticationTypes = [ "/\=SAML\|GUEST", "/admin\=SAML\|basic", "/system\=basic", "/solr\=SAML\|PKI\|basic", "/sources\=SAML\|basic", "/security-config\=SAML\|basic" ]`
+|`complexStringArray = [ "{\"url\"\ \"http://test.sample.com\"\ \"layers\"\ [\"0\"]\ {\"VERSION\"\ \"1.1\|1.2\"\ \"image/png\"}\ \"beta\"\ 1}", "{\"url\"\ \"http://test.sample.com"\ 0.5}", "/solr\=SAML\|PKI\|basic", "/security-config\=SAML\|basic" ]`
```

Comment:
@stustison This got changed as the example in the documentation does not demonstrate how to handle { } and [ ] characters in formatted string arrays. I had to discover for myself whether they required escape characters as double-quotes and whitespaces did. Below this in the documentation is a note on which characters require escapes.